### PR TITLE
[libcurl] Update to 7.85

### PR DESCRIPTION
**Pull request by lugehorsam**
Updates curl from 7.84 to 7.85

Removes normaliz patch as they applied that change to their newest release.

### CODE REVIEW

File: ports/curl/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "curl",
-  "version": "7.84.0",
+  "version": "7.85.0",
   "port-version": 1,
```
Comment (Cheney-W) [inline ports/curl/vcpkg.json:4]:
```suggestion
```

**Comment by lugehorsam** [comment]
@Cheney-W committed your suggestion, thanks.

**Comment by Cheney-W** [comment]
./vcpkg x-add-version libcurl --overwrite-version

**Comment by Cheney-W** [comment]
Could you please solve the conflict? Thank you!

**Comment by Cheney-W** [comment]
Convert this PR to draft since there is no progress. Please ping us if this PR is ready for review again.

**Comment by Cheney-W** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**